cnames_active.js

```diff
@@ -1408,7 +1408,7 @@ var cnames_active = {
   "itemszop": "michaljaz.github.io/itemszop",
   "itsashis4u": "itsashis4u.github.io", // noCF? (don´t add this in a new PR)
   "itunes-bridge": "angrykiller.github.io/iTunes-bridge",
-  "itzik": "itzik.netlify.app",
+  "itzik": "cname.vercel-dns.com",
```

Comment:
```suggestion
  "itzik": "cname.vercel-dns.com", // noCF
```
👋 As you're looking to use Vercel, you'll want to have the Cloudflare proxy disabled

Reply:
Thanks @MattIPv4